clang/runtime/dpct-rt/include/util.hpp.inc

```diff
@@ -161,230 +161,6 @@ inline void matrix_mem_copy(T *to_ptr, const T *from_ptr, int to_ld,
 }
 // DPCT_LABEL_END
 
-// DPCT_LABEL_BEGIN|cast_double_to_int|dpct
```

Comment:
How about:
1. add `namespace math` for all functions in math.hpp
2. keep interfaces here to call dpct::math::...